cromwell.example.backends/LocalExample.conf

```diff
@@ -92,18 +92,25 @@
 
             # Call caching strategies
             caching {
-              # When copying a cached result, what type of file duplication should occur. Attempted in the order listed below:
+              # When copying a cached result, what type of file duplication should occur.
+              # possible values: "hard-link", "soft-link", "copy", "cached-copy".
+              # For more information check: https://cromwell.readthedocs.io/en/stable/backends/HPC/#shared-filesystem
+              # Attempted in the order listed below:
               duplication-strategy: [
                 "hard-link", "soft-link", "copy"
               ]
 
-              # Possible values: file, path, path+modtime
-              # "file" will compute an md5 hash of the file content.
+              # Possible values: md5, xxh64, fingerprint, path, path+modtime
+              # For extended explanation check: https://cromwell.readthedocs.io/en/stable/Configuring/#call-caching
+              # "md5" will compute an md5 hash of the file content.
+              # "xxh64" will compute an xxh64 hash of the file content. Much faster than md5
+              # "fingerprint" will take last modified time, size and hash the first 10 mb with xxh64 to create a file fingerprint.
+              # This strategy will only be effective if the duplication-strategy (above) is set to "hard-link", as copying changes the last modified time.
               # "path" will compute an md5 hash of the file path. This strategy will only be effective if the duplication-strategy (above) is set to "soft-link",
               # in order to allow for the original file path to be hashed.
               # "path+modtime" will compute an md5 hash of the file path and the last modified time. The same conditions as for "path" apply here.
-              # Default: file
-              hashing-strategy: "file"
+              # Default: "md5"
+              hashing-strategy: "fingerprint"
```

Comment:
I'm not sure how literally users "adapt" these example configs so I'm not sure about changing the choice here.

Reply:
Changed it back to md5.